src/test/java/net/masterthought/cucumber/generators/integrations/FeatureReportPageIntegrationTest.java

```diff
@@ -425,8 +405,8 @@ private static void validateHook(HookAssertion[] hookAssertions, Hook[] hooks, S
         }
     }
 
-    private void assertEmbeddingFileExist(Embedding embedding) {
-        File file = new File(configuration.getEmbeddingDirectory(), embedding.getFileName());
-        assertThat(file).exists();
-    }
+//    private void assertEmbeddingFileExist(Embedding embedding) {
```

Comment:
unused method should be removed, not commented out

git keeps history of changes